UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -280,7 +285,23 @@ private async Task CheckProductOrCancellationData(SearchBatchResponse internalSe
                 internalSearchBatchResponse.Entries.Add(item);
                 productList.Add(compareProducts);
                 if (CommonHelper.IsPeriodicOutputService)
-                    await PerformBatchFileDownloadForLargeMediaExchangeSet(item, productItem, exchangeSetRootPath, message, cancellationTokenSource, cancellationToken);
+                {
+                    if (aioConfiguration.IsAioEnabled)
+                    {
+                        if (!aioCells.Contains(productItem.ProductName))
+                        {
+                            await PerformBatchFileDownloadForLargeMediaExchangeSet(item, productItem, exchangeSetRootPath, message, cancellationTokenSource, cancellationToken);
+                        }
+                        else
+                        {
+                            await PerformBatchFileDownload(item, productItem, exchangeSetRootPath, message, cancellationTokenSource, cancellationToken);
+                        }
+                    }
+                    else
+                    {
+                        await PerformBatchFileDownloadForLargeMediaExchangeSet(item, productItem, exchangeSetRootPath, message, cancellationTokenSource, cancellationToken);
+                    }
+                }
                 else
```

Comment:
Not your change but can you add brackets around this for the scope of this else

Reply:
Resolved